css/style.css

```diff
@@ -0,0 +1,66 @@
+body {
+  margin: 0;
+  padding: 5%;
+  color: #172b4d;
+  background-image: url("/images/header-illsutration-mobile@2x.png");
+}
+
+.header {
+  /* font-family: 'Inter'; */
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  text-align: center;
+}
```

Comment:
- [ ] Please from the project requirements, you need to lay out the elements on the page and you should use Flexbox to display this section. The content should be aligned with flexbox. This gives us a lot of options to align all our elements, plus, it's responsive. :blush: